tests/end_to_end_tests.py

```diff
@@ -40,6 +40,20 @@ def test_mnist(self):
 
         assert min_test_score > 0.95
 
+    def test_titanic(self):
+        """
+        Test performance on titanic dataset
+        """
+        # run test
+        subprocess.run(["python", "main.py", "-f", "tests/config_end_to_end_test_titanic.yml"])
+
+        df = Test_EndToEndTest.get_latest_dataframe("*end_to_end_test*")
+
```

Comment:
Is it possible to use the package version of the library ?
Like : 
```python
from mplc.scenario import Scenario

scenario1 = Scenario(2, [0.4,0.6], epoch_count=3, minibatch_count=4)
scenario1.run()
assert  min(scenario1.mpl.test_score) > 0.6 
```

Reply:
Oh yes indeed!